src/pages/Pool-V3/hooks/useCreatePositionForm.ts

```
@@ -0,0 +1,595 @@
+    if (list[i].lowerTick <= target && list[i].upperTick >= target) {
+      return Number(list[i].liquidityAmount.toString());
+    }
+  }
+  return 0;
+}
+
+function numberExponentToLarge(numIn) {
+  numIn += ""; // To cater to numric entries
+  var sign = ""; // To remember the number sign
+  // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+  numIn.charAt(0) == "-" && ((numIn = numIn.substring(1)), (sign = "-")); // remove - sign & remember it
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)